### ISSUE TITLE
SSL error while downloading CLI update

### ISSUE THREAD

**Issue by jimmyjsk** [wakatime/visualstudio-wakatime#74]
Hello, for a while now I have trouble downloading CLI updates. Ive done it manualy for previous version, but it has come up again.

The error in logs says:
```
[Wakatime Info 10:46:34 dopoludnia] Initializing WakaTime v4.0.2
[Wakatime Info 10:46:34 dopoludnia] Current wakatime-cli version is 10.1.3
[Wakatime Info 10:46:34 dopoludnia] Checking for updates to wakatime-cli...
[Wakatime Info 10:46:34 dopoludnia] Found an updated wakatime-cli v10.2.1
[Wakatime HandledException 10:46:35 dopoludnia] Are you behind a proxy? Try setting a proxy in WakaTime Settings with format https://user:pass@host:port. Exception Traceback:: System.Net.WebException: The request was aborted: Could not create SSL/TLS secure channel.
   at System.Net.WebClient.DownloadFile(Uri address, String fileName)
   at System.Net.WebClient.DownloadFile(String address, String fileName)
   at WakaTime.Dependencies.DownloadAndInstallCli()
   at WakaTime.WakaTimePackage.InitializeAsync()
[Wakatime Info 10:46:35 dopoludnia] Finished initializing WakaTime v4.0.2
```

The critical part is:
`Could not create SSL/TLS secure channel.`

I am sure I am not behind a proxy, and when I download  the CLI manualy with a browser from 
`CliUrl = "https://github.com/wakatime/wakatime/archive/master.zip";` it works

My theory is, there is a mismatch in TLS encryption when trying to download the CLI update trough .NET WebClient. I have looked at visual studio plugin, you are targeting v4.5.2 .net framework, which doesnt use TLS 1.2 natively, you have to opt-in ( https://blogs.perficient.com/microsoft/2016/04/tsl-1-2-and-net-support/ ). And github uses only TLS 1.2 encryption on its servers.

Can you please look at it and confirm my suspicions?

**Comment by alanhamlett** [comment]
Thanks for figuring this out! It should be fixed with 5d3c1b601cc1ce3639d810c86017816ccf7d1bf0, and released in v8.0.7.

**Comment by jimmyjsk** [comment]
Thanks :-)
